client/ayon_core/hosts/max/api/lib.py

```diff
@@ -242,20 +244,19 @@ def get_frame_range(folder_entiy=None) -> Union[Dict[str, Any], None]:
         dict: with frame start, frame end, handle start, handle end.
     """
     # Set frame start/end
-    if folder_entiy is None:
-        folder_entiy = get_current_project_folder()
-
-    folder_attributes = folder_entiy["attrib"]
-    frame_start = folder_attributes.get("frameStart")
-    frame_end = folder_attributes.get("frameEnd")
+    if task_entity is None:
+        task_entity = get_current_task_entity()
+    task_attributes = task_entity["attrib"]
+    frame_start = task_attributes.get("frameStart")
+    frame_end = task_attributes.get("frameEnd")
 
     if frame_start is None or frame_end is None:
         return {}
```

Comment:
These should really exist on AYON entities, if not - then that's a bug and should be crashed on as far as I know.
```suggestion
    frame_start = task_attributes["frameStart"]
    frame_end = task_attributes["frameEnd"]
```